zigpy_deconz/config.py

```diff
@@ -26,13 +26,20 @@
 CONF_WATCHDOG_TTL = "watchdog_ttl"
 CONF_WATCHDOG_TTL_DEFAULT = 600
 
+CONF_DEVICE_BAUDRATE = "baudrate"
+
+SCHEMA_DEVICE = SCHEMA_DEVICE.extend(
+    {vol.Optional(CONF_DEVICE_BAUDRATE, default=115200): int}
```

Comment:
The default baudrate I think should remain 38400, as changing it globally will break existing installs.

Instead, we should update `ControllerApplication.probe` to try both baudrates and return the correct one.

Reply:
I agree with the default baudrate change, it should be 38400  (Raspbee I/II and ConBee I run on 38400, the ConBee II doesn't care about the used baudrate, as it uses a USB-CDC driver). Could you provide me with a reference for `ControllerApplication.probe`, so i can research this ?

BTW: Check your Discord DMs we (phoscon) want to send you a ConBee III free of charge